presto-main/src/main/java/com/facebook/presto/SystemSessionProperties.java

```diff
@@ -1401,6 +1402,15 @@ public SystemSessionProperties(
                         "The native engine executable file path for native engine execution",
                         featuresConfig.getNativeExecutionExecutablePath(),
                         false),
+                new PropertyMetadata<>(
+                        NATIVE_EXECUTION_TASK_REQUEST_TIMEOUT,
+                        "The native engine task request max timeout time",
```

Comment:
what request? Is it http request or the total request time? Could we clarify on it a bit?